README.md

```diff
@@ -139,7 +139,7 @@ Schedule C and Schedule D itemizations. Also, passing
 ``{'filter_itemizations': []}`` to ``options`` will result in only the header
 and the filing being parsed and returned.
 
-Including `{'as_strings': True}` in the `options` dictionary will not attempt to convert values that are normally numeric or datetimes to their native python types and will return dictionaries with all values as strings.
+Including `{'as_strings': True}` in the `options` dictionary will not attempt to convert values that are normally numeric or datetimes to their native python types and will return dictionaries with all values as strings. Including `{'simple_conversion': True}` will bypass int conversion and convert datetimes to strings that look like `YYYY-MM-DD` (from `YYYYMMDD`), which speeds up conversion significantly.
```

Comment:
Is there a reason not to just use `as_strings` and then change the date structure generally to `YYYY-MM-DD`?

Reply:
Not a great reason but there was some special float handling and `as_strings` doesn't include it

Reply:
What happens if we treat the floats as strings also?

Reply:
I think that would be fine actually! I just wasn't doing it since it was causing a diff in the output CSV, likely because it might change the number of decimals in the string representation or something like that. As long as you also think it would be good!

Reply:
hmm yeah, I think that would be fine too then. Especially since we are saving everything as `Decimal(12, 2)` in Postgres anyway. Though I am a bit unsure what this might mean for athena, something for us to look out for.

Reply:
Is the speed-up from that change substantial?